core/tests/puppeteer-acceptance-tests/puppeteer-testing-utilities/test-constants.js

```diff
@@ -39,7 +39,7 @@ let testConstants = {
     blogPostThumbnailImage: path.resolve(
       __dirname, '../images/blog-post-thumbnail.svg')
   },
-  DEFAULT_SPEC_TIMEOUT: 300000
+  DEFAULT_SPEC_TIMEOUT: 480000
```

Comment:
Why so long?

Reply:
The blog-admin tests were failing due to timeout for the earlier one. But since then it's been fixed. I can try to again switch the time limit and see if it works still.